### PR TITLE
chore: update std version

### PR DESCRIPTION
**Pull request by irbull**
Update the std library version to 0.224.0 the last version of the std library hosted on deno.land/x.

https://github.com/denoland/deno-astro-adapter/issues/26

### CODE REVIEW

File: src/index.ts [hunk 1]
```diff
@@ -71,9 +71,8 @@ const COMPATIBLE_NODE_MODULES = [
 // replaced with the Deno-specific contents listed below.
 const DENO_IMPORTS_SHIM = `@deno/astro-adapter/__deno_imports.ts`;
 const DENO_IMPORTS =
-  `export { Server } from "https://deno.land/std@${DENO_VERSION}/http/server.ts"
-export { serveFile } from 'https://deno.land/std@${DENO_VERSION}/http/file_server.ts';
-export { fromFileUrl } from "https://deno.land/std@${DENO_VERSION}/path/mod.ts";`;
+  `export { Server, serveFile } from "jsr:@std/http@${STD_VERSION}"
```
Comment (kt3k) [inline src/index.ts:74]: This increases the amount of downloaded modules. Can you split this into 2 imports?:

```suggestion
  `export { Server } from "jsr:@std/http@${STD_VERSION}/server";
export { serveFile } from "jsr:@std/http@${STD_VERSION}/file-server";
```

Comment (irbull) [reply src/index.ts]: Done. I forced pushed it to the branch to keep the 2 commits intact (1 commit for updating the version of the std, the other one for using JSR).

Comment (kt3k) [reply src/index.ts]: Thanks!

**Review by kt3k — APPROVED** [review]
Checked the build using this branch locally. It seemed working. Thanks for your contribution!

**Comment by kt3k** [comment]
Published as 0.1.4.

The imports in `dist/server/entry.mjs` look like the below, and seem working:

Screenshot 2024-10-23 at 14 48 01

**Comment by kt3k** [comment]
I was also able to deploy it to Deploy

repo: https://github.com/kt3k/deno-astro-adapter-0.1.4-test
deploy: https://yoshiyahino-deno-astro-38.deno.dev/

**Pull request merged**